remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,20 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestFlattenMap(t *testing.T) {
+	testString := []byte(`{"name":"shoubhik","assignee":{"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
+	var nestedMap map[string]interface{}
+	err := json.Unmarshal(testString, &nestedMap)
+
+	if err != nil {
```

Comment:
The input is part of the test data (hand-crafted) and `json.Unmashall` is part of the std. library, so this assertion is not necessary.